engine/runner/src/main/scala/org/enso/runner/Main.scala

```diff
@@ -1056,9 +1056,13 @@ object Main {
         exitFail()
     }
 
-  /** Default log level to use if the LOG_LEVEL option is not provided.
+  /** Default log level to use when starting a language server if the
+    * `LOG_LEVEL` option is not provided.
     */
-  private val defaultLogLevel: Level = Level.WARN
+  private val defaultServerLogLevel: Level = Level.WARN
+
+  /** Default log level to use if the `LOG_LEVEL` option is not provided. */
+  private val defaultLogLevel: Level = Level.ERROR
```

Comment:
Is it wise to change log level for all messages when the goal is to suppress single message?

Reply:
I only switched it back to its original value https://github.com/enso-org/enso/pull/7825/files#diff-7111bba91c7d387c5ae7e93bc67c9e0b513b31f99e3a10a86d1a1cff82baff0b

Reply:
I think it should stay as `WARN`. Just the level of the offending log message should be lowered.

Reply:
I see. Why do I have a feeling we are shuffling logging and its configuration back and forth without any serious plan?

Reply:
> I see. Why do I have a feeling we are shuffling logging and its configuration back and forth without any serious plan?

I don't know. I don't have such a feeling.